po/ru.po

```diff
@@ -4356,6 +4356,19 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"```rust,editable\\n\n"
```

Comment:
I believe `\\n\n` should be just `\n` here and below.

Reply:
Duh! I had to manually fix the merge and copied too many backslashes. :) Let me fix that.

Reply:
Thanks!